torch_xla/experimental/megablox/__init__.py

```diff
@@ -0,0 +1 @@
+from .gmm import gmm

```

Comment:
Once we remove all the duplicated code. We can move this method back to custom_kernel.py.